ocp_resources/node_health_check.py

```diff
@@ -0,0 +1,10 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class NodeHealthCheck(NamespacedResource):
```

Comment:
Should this be namespacedResource or Resource? from https://github.com/medik8s/node-healthcheck-operator, it does not look like it is associated with a namespace.

Reply:
https://docs.openshift.com/container-platform/4.10/nodes/nodes/eco-node-health-check-operator.html#about-node-health-check-operator_node-health-check-operator seems like it is. @geetikakay ?

Reply:
It’s does have a default namespace which openshift-operator. I have removed the link to downstream documentation. Also it does allow to link it up with a user defined namespace.

Reply:
@rnetser yes thanks this is the one

Reply:
I wanted to add a link to documentation so that we know where you get understanding of this feature and future development in this area. I don’t wanted to share downstream links